randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -404,61 +412,88 @@ Extra - room_id: [13]
   * Layers: default
   * L0 Hatch to Mimic Storage/Door to Frozen Tower
   * Extra - door_idx: (24,)
+  > Door to Data Room
+      All of the following:
+          Space Jump
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Hi-Jump and Damage Runs (Advanced) and Cold Damage ≥ 238
   > Bottom of Room
-      # TODO: dmg
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 265
 
 > Door to Groznyj Shaft; Heals? False
   * Layers: default
   * L3 Hatch to Groznyj Shaft/Door to Frozen Tower
   * Extra - door_idx: (49,)
   > Bottom of Room
-      # TODO: dam
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 16
 
 > Other to Transmutation Trial; Heals? False
   * Layers: default
   * Tunnel to Transmutation Trial/Other to Frozen Tower
   * Extra - door_idx: (79,)
   > Bottom of Room
-      # TODO: dmg
-      Varia Suit
+      Any of the following:
+          # Damage Reqs
+          Varia Suit
+          Damage Runs (Advanced) and Cold Damage ≥ 17
 
 > Door to Data Room; Heals? False
   * Layers: default
   * L3 Hatch to Data Room/Door to Frozen Tower
   * Extra - door_idx: (90,)
   > Door to Mimic Storage
       All of the following:
-          # TODO: dmg
-          Varia Suit
+          Any of the following:
+              # Damage Reqs
+              Varia Suit
+              Space Jump and Damage Runs (Advanced) and Cold Damage ≥ 59
           # Drop down from ledge above https://youtu.be/lUc4sT4UyQY
           Jump Extends (Expert) and Movement (Hypermode)
```

Comment:
this doesnt make much sense. it will still require the jump extend despite you having SJ.